module/Finna/src/Finna/RecordDriver/SolrEad3.php

```diff
@@ -392,6 +394,36 @@ public function getOriginationExtended(): array
         return $localeResults ?: $results;
     }
 
+    /**
+     * See if holdings tab is shown on current item
+     *
+     * @return bool
+     */
+    public function allowRequestForm()
+    {
+        $xml = $this->getXmlRecord();
+        $datasourceSettings = $this->datasourceSettings[$this->getDataSource()];
+        if ($xml && $datasourceSettings) {
+            // Requests only allowed on specified datasource
+            if (
+                $datasourceSettings['allowArchiveRequest']
+                && !empty($this->getFilingUnit())
```

Comment:
Onko tämäkin sellainen vaatimus, että pitäisi olla asetuksissa?

Reply:
Kyseessä on tietueen kenttä (container), joten ei sinällään kuulu tietolähteeseen? Eli osaa sanoa kuuluisiko se sitten enemmänkin asetuksiin

Reply:
Tarkoitin sitä, että pitäisi ehkä olla asetuksissa vaaditaanko, että filing unit on määritelty.

Reply:
Jaa, eli olisi myös mahdollista varata aineistoa, joilla ei olisi kenttää, jos näin on päätetty tehdä/sallia.